3.processing_CP_features/extract_sc_features.ipynb

```diff
@@ -0,0 +1,2827 @@
+{
```

Comment:
Can you create an `nbconverted` .py file?

Reply:
(and add to a `3.processing_CP_features/scripts/nbconverted folder`)

Reply:
:+1: